web/pom.xml

```diff
@@ -838,7 +838,7 @@
           <contextXml>${basedir}/jetty-context.xml</contextXml>
           <webAppSourceDirectory>${project.build.directory}/geonetwork</webAppSourceDirectory>
```

Comment:
does this line need any changes?

Reply:
It was working fine because the build is still done in the same folder see `webappDirectory` https://github.com/geonetwork/core-geonetwork/blob/master/web/pom.xml#L831